doc/source/user_guide/fields/reduction.rst

```diff
@@ -3,350 +3,302 @@
 Reduction
 =========
 
-You can use reduction functions on Fluent data from one
-or across multiple remote Fluent sessions.
+You can use reduction functions on Fluent data from one or across multiple remote Fluent sessions.
+PyFluent provides both **functional** and **object-oriented** approaches to applying reduction functions.
+While both are supported, the **functional approach** is emphasized for its flexibility,
+particularly when working with multiple solver sessions.
 
-Accessing reduction functions
------------------------------
+Introduction to Reduction Functions
+-----------------------------------
 
-In order to access reduction function, import it and launch the Fluent solver.
-Then, make boundary conditions data, etc. available (for example, by reading case files):
+Reduction functions perform operations like computing averages, integrals, and sums over specified data locations,
+such as areas or volumes.
+
+For demonstrating the following examples, let us initialize two separate solver sessions
+with two separate examples case files as follows:
 
 .. code-block:: python
 
-  >>> import ansys.fluent.core as pyfluent
-  >>> from ansys.fluent.core.examples import download_file
-  >>> solver = pyfluent.launch_fluent(mode=pyfluent.FluentMode.SOLVER)
-  >>> case_path = download_file("Static_Mixer_main.cas.h5", "pyfluent/static_mixer")
-  >>> solver.settings.file.read(file_type="case", file_name=case_path)
+    >>> import ansys.fluent.core as pyfluent
+    >>> from ansys.fluent.core.solver.function import reduction
+    >>> from ansys.fluent.core.examples import download_file
 
+    >>> solver1 = pyfluent.launch_fluent(mode=pyfluent.FluentMode.SOLVER)
+    >>> case_path = download_file("Static_Mixer_main.cas.h5", "pyfluent/static_mixer")
+    >>> solver1.settings.file.read(file_type="case", file_name=case_path)
```

Comment:
Please use concise form of the API whereever possible:
```
solver1 = pyfluent.launch_fluent()
solver1.settings.file.read_case(file_name=...)
```

Reply:
Sure, done, Thank you.